src/styles/_catalog.scss

```diff
@@ -1,22 +1,36 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
```

Comment:
```suggestion
    grid-template-columns: repeat(2, 200px);
```

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/206854922-a6fbb57a-e41e-49c2-824e-9cfd93ee1e28.png)